src/index.js

```diff
@@ -1,5 +1,16 @@
-const trialDivision = (number) => {
-    
+const trialDivision = (n) => {
+
+  let i = 2;
+  if ( n <= 1 || n % 1 !== 0 ) {
+      return false;
+  }
+  while ( i < n ) {
+      if (n % i === 0) {
+          return false;
+      }
+      i += 1;
   }
```

Comment:
Si sabes el número de veces que se realizará el ciclo, es mejor usar un for loop